adapters/openx/openxtest/supplemental/multi-imp.json

```diff
@@ -7,7 +7,7 @@
         "banner": {},
         "ext": {
           "bidder": {
-            "unit": "banner-unit-1",
+            "unit": "111",
```

Comment:
Curious, why "111" instead of "1"? Is there a test involved with the length of the unit string?

Reply:
No functional reason, just think it's easier to find and replace if needed.